src/app/common/error-handlers/toaster-error-handler.ts

```diff
@@ -1,7 +1,8 @@
 import { RavenErrorHandler } from './raven-error-handler';
```

Comment:
I think you forgot to remove the RavenErrorHandler.

Reply:
fixed